postgres-appliance/scripts/configure_spilo.py

```diff
@@ -376,7 +376,7 @@ def get_instance_metadata(provider):
     headers = {}
     if provider == PROVIDER_GOOGLE:
         headers['Metadata-Flavor'] = 'Google'
-        url = 'http://metadata.google.internal/computeMetadata/v1/instance'
+        url = 'http://169.254.169.254/computeMetadata/v1/instance'  # metadata.google.internal
```

Comment:
should this be env var?

Reply:
I thought about it, but decided against: to me it seems readability, being able to see the whole url at once, trumps the benefits of defining the base url in a one place only in this case, especially given that the address is very unlikely to change.